src/partials/footer.html

```diff
@@ -1,26 +1,28 @@
 <footer>
   <section class="footer container">
-    <div class="footer__title">
-      <p class="footer__title--light">Homemade ice cream made</p>
-      <p class="footer__title--bold">with passion</p>
-    </div>
-    <div class="footer__social-links">
-      <a class="footer__social-icon" href="#">
-        <svg width="32px" height="32px">
-          <use href="*"></use>
-        </svg>
-      </a>
-      <a class="footer__social-icon" href="#">
+    <h2 class="footer__title">
+      Homemade ice cream made <span class="footer__title--bold">with passion</span>
```

Comment:
jeśli używasz mixin niestety w klasie musimy to zapisać tak: "footer__title footer__title--bold" nie możemy pomijać rdzenia

Reply:
Zupełnie o tym zapomniałam! Jak dobrze, że czuwasz. Poprawione!